wrench/src/wrench.rs

```diff
@@ -240,6 +242,16 @@ impl Wrench {
         wrench
     }
 
+    pub fn get_page_zoom(&self) -> ZoomFactor {
+        self.page_zoom_factor
+    }
+
+    pub fn set_page_zoom(&mut self, zoom_factor: ZoomFactor) {
+        self.page_zoom_factor = zoom_factor;
+        self.api.set_page_zoom(self.document_id, self.page_zoom_factor);
+        self.set_title("");
```

Comment:
Does setting "extra" to the empty string clobber anything? Or is it basically dead code?

Reply:
The extra string is only used during binary replay, so although it's not a very tidy API, it shouldn't have any bad side-effects.